src/test/resources/test.template.properties

```diff
@@ -37,7 +37,9 @@ test.firefox.path=
 
 ###############################################################################
 # Given below are the test accounts/passwords used for testing. 
-# The values given below will work for the dev server.
+# The values given below will work for the dev server(except one test case GodModTest).
+# To pass GodModTest, change ALL the accounts(test.admin.account) by appending to the back of it.
```

Comment:
Isn't this supposed to be "... change ALL the accounts by appending ..." without mentioning "(test.admin.account)"?